src/aleph/ccn_cli/services/keys.py

```diff
@@ -0,0 +1,43 @@
+import os.path
+
+from p2pclient.libp2p_stubs.crypto.rsa import (
+    KeyPair,
+    create_new_key_pair,
+)
+
+
+def generate_keypair() -> KeyPair:
+    """
+    Generates a new key pair for the node.
+    """
+    key_pair = create_new_key_pair()
+    return key_pair
```

Comment:
Why this proxy function ?

```suggestion
generate_keypair =  create_new_key_pair
```